doc/nrf/app_dev/device_guides/nrf54l/vpr_flpr.rst

```diff
@@ -0,0 +1,29 @@
+.. _vpr_flpr_nrf54l:
+
+Working with the FLPR core
+##########################
+
+The nRF54L15 SoC has a dedicated RISC-V CPU (VPR), a fast lightweight peripheral processor (FLPR) designed for software-defined peripherals (SDP).
+This core currently supports the following peripherals:
```

Comment:
Which CPU supports these peripherals? Are SDPs emulating these peripherals available? Coes Application CPU support these? Of are device drivers for these available in FLPR? Or something else :)

Reply:
How about
`Following peripherals are available to be utilized on the FLPR core, using appropriate Zephyr Device Driver API:`